### DISCUSSION TITLE
Is there a way to check if Window is Maximized in a frameless application  when drag "drag handler"?

### DISCUSSION THREAD

**Discussion by buzheng** [wailsapp/wails#3598] [Q&A]
There are two ways to change Window Maximize in windows 11:

1. Drag the "drag handler" to the top of screen, the window will be maximized.
2. When the window is maximized, dragging the "drag handler" will let the window be normal.

I want to update the Maximize Button immediately, but I have no idea when and where to check if the window is maximize. Please help me, thanks very much.

**Comment by levinit** [comment]
maybe you can listen event such as:
fullscreen event https://developer.mozilla.org/en-US/docs/Web/API/Document/fullscreenchange_event 
resize envent  https://developer.mozilla.org/en-US/docs/Web/API/Window/resize_event

and
window.fullscreen  property https://developer.mozilla.org/en-US/docs/Web/API/Window/fullScreen
